### PR TITLE
Calculate initial height properly

### LINKED ISSUE #8527: initialRows/initialColumns still doesn't work quite right
By default WT starts with `"initialRows": 30', and this is true when there is not the tab row. After opening the first tab, tha tab row shows up and WT really displays 28 rows (it conserves the window height). 

Is there an option to conserve the `initiaRows', 30, for example? (the window height should increase a bit).

### PR DESCRIPTION
**Pull request by Hegunumo**
Closes #8527

### CODE REVIEW

**Review by zadjii-msft — CHANGES_REQUESTED** [review]
Okay, I don't think this is the right solution. First and foremost, I think we should already be handling this case in the `if`/`else if` below this. That's probably the thing that's returning the wrong value. 

Secondly - `TermControl` shouldn't need to know anything about the presence of tabs or not. It should be `TerminalPage`/`AppLogic`/`TerminalApp` responsibility to account for the size of the tabs, not the control. The control should be agnostic of whoever's hosting it (it doesn't care if the hosting application has tabs or not)

**Comment by Hegunumo** [comment]
I tested all four true false cases, and it worked. However I'm not sure this was the right solution.

File: src/cascadia/TerminalApp/AppLogic.cpp [hunk 1]
```diff
@@ -594,7 +594,7 @@ namespace winrt::TerminalApp::implementation
             // For whatever reason, there's about 6px of unaccounted-for space
             // in the application. I couldn't tell you where these 6px are
             // coming from, but they need to be included in this math.
```
Comment (zadjii-msft) [inline src/cascadia/TerminalApp/AppLogic.cpp:596]:
```suggestion
            // For whatever reason, there's about 10px of unaccounted-for space
            // in the application. I couldn't tell you where these 10px are
            // coming from, but they need to be included in this math.
```

**Review by zadjii-msft — CHANGES_REQUESTED** [review]
I can't believe all this time, it said `Width` and not `Height` 🤦‍♂️ 

I'm only blocking over updating the comment, but yea the fix seems good to me.

**Comment by Hegunumo** [comment]
@zadjii-msft I fixed the comment.

**Comment by DHowett** [comment]
Wow. I can't believe we were calculating the width instead of the height. Thank you!

**Review by DHowett — APPROVED** [review]

**Review by zadjii-msft — APPROVED** [review]

**Pull request merged**